src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -51,13 +51,13 @@
 #endif
 
 #ifdef TARGET_TX_ESP32_E28_SX1280_V1
-#define MaxPower 4
-#define DefaultPowerEnum 2
+#define MaxPower PWR_500mW
+#define DefaultPowerEnum PWR_50mW
 #endif
 
 #ifdef TARGET_TX_ESP32_LORA1280F27
-#define MaxPower 4
-#define DefaultPowerEnum 2
+#define MaxPower PWR_500mW
```

Comment:
LORA1280F27 can only do 250.  And in reality is really crap and should be deleted.

Reply:
I don't know the real output power but searching on google I always find 500mW. Doesn't "27" in LORA1280F**27** stands for 27dbm (500mW)?

Reply:
They can rate it even 1W but reality is something else 😂
That module is bad as @JyeSmith already mentioned... It will push over that limit anyway :)

Reply:
Ok, I understood I should never buy this module 😂
So in conclusion, what max power should I set? 250mW?

Reply:
Keep what is was so 250mW